website/docs/guides/customize-schema-alias.md

```diff
@@ -0,0 +1,521 @@
+
+- In dbt Cloud, we recommend utilizing [environment variables](/docs/build/environment-variables) to define where the dbt invocation is occurring (dev/stg/prod).
+
+- They can be set at the environment level and all jobs will automatically inherit the default values. We'll add jinja logic (`if/else/endif`) to identify whether the run happens in dev, prod, Ci, and more.
+    
+- Or as an alternative to environment variables, you can use `target.name`. For more information, you can refer to [About target variables](/reference/dbt-jinja-functions/target). 
+
+
+<Lightbox src="/img/docs/dbt-cloud/using-dbt-cloud/Environment Variables/custom-schema-env-var.png" title="Custom schema environmental variables target name." />
+
+To allow the database/schema/object name to depend on the current branch, you can use the out of the box `DBT_CLOUD_GIT_BRANCH` environment variable in dbt Cloud [special environment variables](/docs/build/environment-variables#special-environment-variables).
+
+
+## Example use cases
+
+Here are some typical examples we've encountered with dbt users leveraging those 3 macros and different logic. 
+
+:::note
+
+Note that the following examples do not comprehensively cover all the available options. These examples are meant to be templates for you to develop your own behaviors. 
+
+:::
+
+- [Use custom schema without concatenating target schema in production](/guides/customize-schema-alias?step=3#1-custom-schemas-without-target-schema-concatenation-in-production)
+- [Adding developer identities to tables](/guides/customize-schema-alias?step=3#2-static-schemas-adding-developer-identities-to-tables)
+- [Using branch name as schema prefix](/guides/customize-schema-alias?step=3#3-using-branch-name-as-schema-prefix)
+- [Using a static schema for CI](/guides/customize-schema-alias?step=3#4-using-a-static-schema-for-ci)
+
+### 1. Custom schemas without target schema concatenation in production
+
+
+The most common use case is using the custom schema without concatenating it with the default schema name when in production.
+
+To do so, you can create a new file called `generate_schema_name.sql` under your macros folder with the following code:
+
+recommend adding file name for the jinja code (and all codes) so users know the file path:
+
+<File name='macros/generate_schema_name.sql'>
+```jinja
+
+{% macro generate_schema_name(custom_schema_name, node) -%}
+
+    {%- set default_schema = target.schema -%}
+    {%- if custom_schema_name is none -%}
+
+        {{ default_schema }}
+
+    {%- elif  env_var('DBT_ENV_TYPE','DEV') == 'PROD' -%}
+        
+        {{ custom_schema_name | trim }}
+
+    {%- else -%}
+
+        {{ default_schema }}_{{ custom_schema_name | trim }}
+
+    {%- endif -%}
+
+{%- endmacro %}
+
+
+
+        
+
+
+
+
+
+```
+</File>
+This will generate the following outputs for a model called `my_model` with a custom schema of `marketing`, preventing any overlap of objects between dbt runs from different contexts.
```

Comment:
```suggestion


This will generate the following outputs for a model called `my_model` with a custom schema of `marketing`, preventing any overlap of objects between dbt runs from different contexts.
```